src/components/Cards.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable no-unreachable */
+/* eslint-disable no-undef */
+import React, { useState, useEffect } from 'react';
+import CardItem from './CardItem';
+import { useParams } from 'react-router-dom';
+import './Cards.css'
+
+function Cards() {
+    const { id } = useParams()
+const [pet, setPet] = useState([]);
+const [loading, setLoading] =useState(false)
+
+const fetchPets = async () => {
+        setLoading(true)
+        try {
+            const response = await fetch(`https://pets-v2.dev-apis.com/pets?id=${id}`);
+            const data = response.json();
+            setPet([data.pets]);
+            setLoading(false);
+            
+        }catch (error){
+            alert(error.message);
+        }
+         }
+         useEffect(() => {
+            window.scroll(0, 0)
```

Comment:
What does this line do?

Reply:
To call the fetchPets function as soon as the page loads, before the user scrolls.